src/Stakeholders/layouts/Agencies.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+// import UIState from '../../components/UIState';
+import Agencies from '../components/Agencies';
+/**
+ * @function
+ * @name StakeholdersContactsLayout
```

Comment:
Update the name from StakeholdersContactsLayout to StakeholdersAgenciesLayout

Reply:
Okay, I have fixed it